flaskapp/__init__.py

```diff
@@ -15,4 +15,6 @@ def homepage():
         except InvalidRankerException:
             return abort(404)
         else:
-            return render_template("rankers.html", query=query, results=results)
+            return render_template(
+                "rankers.html", query=query, results=results["originalScoreModel"]
```

Comment:
If you just wanted to render the results from the originalScoreModel, you can use the get_results_for_ranker method instead of get_results